spec/support/service_topological_serviceoffering_helper.rb

```diff
@@ -0,0 +1,20 @@
+module ServiceOfferingHelper
```

Comment:
Small nit, renaming this file to `service_offering_topological_service_offering_helper.rb`

Reply:
Done